src/lib/device-os-version-util.js

```diff
@@ -122,13 +122,17 @@ async function isModuleDownloaded(module, version, platformName) {
  * @param {Object} ui - allow us to interact in the console
  * @returns {Promise<*[]>} - true if successful
  */
-async function downloadDeviceOsVersionBinaries({ api, platformId, version='latest', ui }){
+async function downloadDeviceOsVersionBinaries({ api, platformId, version='latest', ui, omitUserPart=false }){
 	try {
 		const downloadedBinaries = [];
 		// get platform by id from device-constants
 		const platform = Object.values(deviceConstants).filter(p => p.public).find(p => p.id === platformId);
 		// get the device os versions
 		const deviceOsVersion = await api.getDeviceOsVersions(platformId, version);
+		if (omitUserPart) {
```

Comment:
I can't imagine why we'd want to download the user part ever. Can you always skip it?

Reply:
I was thinking like the setup,  but specially here there is not a reason to not omit it always Fixed here! https://github.com/particle-iot/particle-cli/pull/657/commits/268d6e892a5a68e954f62c6e0d7e932c8299c4e5